pages/chats/[[id]].vue

```diff
@@ -174,6 +178,24 @@ export default {
     const myid = authStore.user?.id
     const showChats = ref(20)
 
+    const contactDetailsAskModal = ref()
+
+    watch(
```

Comment:
This is an interesting technique.

Let's have a voice call about this, `waitForRef` and async setup.

Reply:
That's the technique I always used and keep using in all my projects. Runs like clockwork. And should definitely be better than a 100ms-setTimeout-based approach. We can have a call, tell me the time.